### PR TITLE
Improves link + button focusing

### PR DESCRIPTION
**Pull request by kenaniah**
**UX Improvements:**
 - [x] Improves `:focus` accessibility for `<a>` links by adding `text-decoration: underline`
 - [x] Removes the blur from buttons after click (as they still have keyboard focus after activation)

### CODE REVIEW

File: components/style/core/base.less [hunk 1]
```diff
@@ -61,6 +61,10 @@ a {
   cursor: pointer;
   transition: color .3s ease;
 
+  &:focus {
+    text-decoration: underline;
```
Comment (afc163) [inline components/style/core/base.less]: `text-decoration: underline` is not fit our design specification.

Comment (kenaniah) [inline components/style/core/base.less]: @afc163 Could you please direct me to where in the specification `text-decoration: underline` is prohibited? 

And if underline is prohibited, do you have another suggestion for visually indicating keyboard focus on link elements?

Comment (afc163) [inline components/style/core/base.less]: Our designer don't wish underline on hover links. But it is focused state in this situation, I need to discuss with them later.

Comment (afc163) [inline components/style/core/base.less]: FYI, you need to run `npm test -- -u` to update demo snapshot, that make ci successd.

Comment (afc163) [inline components/style/core/base.less]: I just talked to our designer and they are OK with your underline suggestion, so I gone merging it now and tweak some underline styles later. Thx for you contribution! 😃

**Comment by ouzhou** [comment]
The underline looks ugly

**Pull request merged**